tests/testit/test-json_test.R

```diff
@@ -0,0 +1,24 @@
+testit::assert("Compares test and ref json for function supplied as function: pass", {
+  test_function <- function(x){ list(names(x)) }
+  out <- capture.output(json_test(func = test_function,
```

Comment:
agree! My goal for this PR was to change as little as possible about how the whole package works while still allowing testing of the main functions, so that if/when we refactor it, we can do so a little bit easier than running tests from within insightrxr and seeing if they still run. 😁